components/admin_console/group_settings/group_details/group_profile.jsx

```diff
@@ -44,8 +44,7 @@ export default class GroupProfile extends React.PureComponent {
                         </div>
                         <input
                             type='text'
-                            id={customID}
-                            className='form-control group_at_mention_input'
+                            className='form-control group-at-mention-input'
```

Comment:
this change is unrelated to the test but it just made sense to change the class to not be snake case since we always use hyphen case for class names